README.md

```diff
@@ -2096,11 +2096,13 @@ Other Style Guides
     const foo = a ? a : b;
     const bar = c ? true : false;
     const baz = c ? false : true;
+    const qux = a ? a : b;
 
     // good
     const foo = a || b;
     const bar = !!c;
     const baz = !c;
+    const qux = a ?? b;
```

Comment:
actually `a ? a : b` is equivalent to `a || b`.

The equivalent to `a ?? b` is `a != null ? a : b`.

Reply:
I changed it. Thank you.